### DISCUSSION TITLE
CrySPY version 1.1.1

### DISCUSSION THREAD

**Discussion by Tomoki-YAMASHITA** [Tomoki-YAMASHITA/CrySPY#50] [Announcements]
### Fixed
- bug fix for gen_struc: delete spg_error

<hr /><em>This discussion was created from the release <a href='https://github.com/Tomoki-YAMASHITA/CrySPY/releases/tag/v1.1.1'>CrySPY version 1.1.1</a>.</em>

**Comment by yw-fang** [comment]
👍👍 Your recent development is made frequently! By the way, do you participate ccp2023 in early August in Kobe. I will present some results about the application of CrySPY, as well as USPEX and CALYPSO if time suffices.

**Reply by Tomoki-YAMASHITA** [reply]
I will pass ccp2023 because I have a thing to do in University during the priod.
Enjoy Japan after a long absence, and promote CrySPY at the meeting!

**Reply by yw-fang** [reply]
Thank you for your words. I am truly glad to have the chance to go back. Actually, I'm also keeping an eye on the potential tenure track positions in Japan. 
Sure, I'll definitly promote CrySPY. In addition, I will post a paper on arXiv this month or next month about the application of CrySPY and CALYPSO into the prediction of Lu-N-H system which was claimed to be a nearly-ambient superconductor by Ranga Dias.

**Reply by Tomoki-YAMASHITA** [reply]
I am looking forward to your paper on the hot topic of the moment.
